lib/command.c

```diff
@@ -127,6 +127,13 @@ bool cmd_allow_reserved_ranges_get(void)
 	return host.allow_reserved_ranges;
 }
 
+const char *cmd_software_version_get(void)
```

Comment:
instead of allocating memory why not do something like this:
```+static const char *stuff_it(void)
+{
+       return FRR_FULL_NAME "/" FRR_VERSION;
+}
+```

Reply:
FIxed.